Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -1083,6 +1096,20 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
 
                 let fileURL = symbolGraphLoader.mainModuleURL(forModule: moduleName)
                 
+                let moduleInterfaceLanguages: Set<SourceLanguage>
+                if FeatureFlags.current.isExperimentalObjectiveCSupportEnabled {
+                    // Infer the module's interface languages from the interface
+                    // languages of the first symbol in the symbol graph.
```

Comment:
Is this comment accurate? Also, do we have a ticket we can link to for SymbolKit to provide an API that gives this information?